### PR TITLE
Blockly: explain the multiselect feature

### PR DESCRIPTION
**Pull request by jimtng**
https://github.com/openhab/openhab-webui/pull/2419

I've removed the cross rule copy paste animated gif for the following reasons:
- It can be summarized in one sentence
- Short animated GIFs are good, but a long one is annoying. You cannot pause them, you cannot rewind / go back. You'd have to painfully wait for it to play out and pay super close attention.

### CODE REVIEW

File: configuration/blockly/rules-blockly-before-using.md [hunk 1]
```diff
@@ -145,14 +156,6 @@ After pressing _Enter_ all found blocks that contain the search term are marked
 
 ![blockly-search](../images/blockly/blockly-search.png)
 
-### Zoom the workspace
```
Comment (stefan-hoehn) [inline configuration/blockly/rules-blockly-before-using.md:148]: I am happy with this PR except that I only notice now that you have seemed to remove the zooming controls or am I wrong in the code PR which I seemed to have overlooked by then?

image

I always liked those and use them quite often.

Comment (jimtng) [reply configuration/blockly/rules-blockly-before-using.md:148]: The zoom controls are still there in the software. In the docs it was moved to the top. Do a search for "zoom".

image

Comment (jimtng) [reply configuration/blockly/rules-blockly-before-using.md:148]: image

**Review by stefan-hoehn — APPROVED** [review]

**Comment by stefan-hoehn** [comment]
Looks good to me, thanks, Jim.

Unfortunately even though the build runs through there seems to be an issue with the deploy which I currently can't find. Until then, I'd rather keep this open.

**Pull request merged**